### ISSUE TITLE
Crash with supplementaries

### ISSUE THREAD

**Issue by killbot24** [p0t4t0sandwich/ModPacketFix#7]
With supplementaries on the client joining a paper server in 1.20.1 will crash the player

**Comment by p0t4t0sandwich** [comment]
Yeah, I noticed that today while working on a modpack, it looks like I'd have to find a way to spoof the datapack entries Supplementaries is expecting
